packages/velo-external-db-core/lib/data_hooks_utils.spec.js

```diff
@@ -3,71 +3,71 @@ const Chance = require('chance')
 const chance = Chance()
 const { Uninitialized } = require('test-commons')
 const { givenBodyWith } = require('../../velo-external-db/test/drivers/hooks_test_support')
```

Comment:
@Idokah  We decided that we would not import things from other packages like that.......